src/controllers/admin.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { IRegisterAdminReqBody } from '../interfaces/controllers/admin';
+import { createAdmin } from '../services/admin.service';
+import { catchAsync } from '../utils';
+import ApiResponse from '../utils/ApiResponse';
+
+const registerAdmin = catchAsync(async (req, res) => {
+    const data: IRegisterAdminReqBody = {
+        name: req.body.name,
+        email: req.body.email,
+        password: req.body.password || '',
+        allow_password_change: req.body.allow_password_change || false,
+    };
+
+    const admin = await createAdmin(data);
+
+    return new ApiResponse(res, 201, 'Admin registered successfully', admin);
```

Comment:
@Dhamareshwarakumar shouldn't we move 201 as well to `statusCodes.ts` file and use it here

Reply:
- Okay missed it.

Reply:
Done